main_page.html

```diff
@@ -5,6 +5,22 @@
     <title>Player cards by Krzysztof</title>
 </head>
 <body>
-
+    <h1 align="center">This is my first madafakin' great project</h1>
```

Comment:
basically it's not recommended to use any styling properties inside HTML. That should be done on the CSS side. So, for now, you can leave it on code but remember during writing CSS to remove them.

Reply:
Communicat from w3school site: The align attribute of <p> is not supported in HTML5. Use CSS instead.